src/jobs/run.yml

```diff
@@ -52,6 +57,10 @@ parameters:
     description: |
       Number of Circle machines to use for load balancing, min 1
       (requires `parallel` and `record` flags in your `cypress-command`)
+  install-browsers:
+    type: boolean
+    default: false
+    description: Flag to install additional browsers to run your tests. This is only needed if you are passing the `--browser`` flag in your `cypress-command`. This includes Chrome, FireFox, ChromeDriver and GeckoDriver. If you need additional browser support you can set this to false and make sure you use an executor with a docker image that includes the browsers of your choosing see https://hub.docker.com/r/cypress/browsers/tags
```

Comment:
Brian had mentioned we don't want GeckoDriver but I imagine it's all or nothing here, right? We may want to omit this and determine what happens with Cypress & GeckoDriver to be aware of issues.

Reply:
The "Install Geck" step literally takes 0 seconds. Not sure if there is value in removing the actual step. But we could probably just not mention it in the documentation?